CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.14.2 (2019-09-17)
+* Metrix: can now be disabled through the configuration
```

Comment:
How do we disable it? Wdyt about adding a 1-liner to describe it?

Reply:
It shouldn't be documented here, because this is the changelog. I'll create another PR to reflect it in the readme of metrix.

Reply:
I meant in the readme yep